pkg/controllers/tagging/tagging_controller.go

```diff
@@ -178,14 +213,9 @@ func (tc *Controller) tagNodesResources(node *v1.Node) error {
 // tagEc2Instances applies the provided tags to each EC2 instance in
 // the cluster.
 func (tc *Controller) tagEc2Instance(node *v1.Node) error {
-	instanceID, err := awsv1.KubernetesInstanceID(node.Spec.ProviderID).MapToAWSInstanceID()
+	instanceID, _ := awsv1.KubernetesInstanceID(node.Spec.ProviderID).MapToAWSInstanceID()
```

Comment:
Error isn't likely/possible?

Reply:
error is not possible as we've already checked and handled it earlier in the process: https://github.com/kubernetes/cloud-provider-aws/pull/334/files#diff-ac633e66d7cfdfa6934f05558a33311d493f704ab69e5443c4ad046a47ae1266R158